Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -917,50 +952,80 @@ public void StartDirtyTracking()
             // throw err if item is serialized but dindn't impl IsDirty
 
             // Properties
-            foreach (PropertyInfo PI in this.GetType().GetProperties())
+            Parallel.ForEach(this.GetType().GetProperties(), PI =>
             {
                 var token = PI.GetCustomAttribute(typeof(IsSerializedForLocalRepositoryAttribute));
-                if (token == null) continue;
+                if (token == null)
+                {
+                    return;
+                }
 
                 DirtyTrackingFields.Add(PI.Name);
 
                 // We track observable list which are seriazlized - drill down recursivley in obj tree
                 if (typeof(IObservableList).IsAssignableFrom(PI.PropertyType))
                 {
+                    //skip list if it is LazyLoad and was not loaded yet
+                    var lazyLoadtoken = PI.GetCustomAttribute(typeof(IsLazyLoadAttribute));
+                    if (lazyLoadtoken != null)
+                    {
+                        string lazyStatusProp = PI.Name + nameof(IObservableList.LazyLoad);
+                        if (this.GetType().GetProperty(lazyStatusProp) != null)
+                        {
+                            if (bool.Parse(this.GetType().GetProperty(PI.Name + nameof(IObservableList.LazyLoad)).GetValue(this).ToString()) == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5012537)